code/style.css

```diff
@@ -0,0 +1,145 @@
+body {
+    font-family: 'Quicksand', sans-serif;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-align: justify;
+    background-image: url("./pizza-pattern.png");
+}
+
+main {
+    display: flex;
+    border-radius: 10px;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+    background: rgb(248, 213, 148);
+}
+
+section {
+    max-width: 550px;
+}
+
+.welcome {
+    border: 5px solid white;
+    padding: 10px;
+    color: black;
+    margin: 10px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    border-radius: 5px;
+}
+
+.cards {
+    border: 5px solid white;
+    padding: 10px;
+    border-radius: 5px;
+    color: black;
+    margin: 10px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+}
+
+.cards h1 {
+    text-align: center;
+}
+
+.menu li {
+    list-style: none;
+    text-align: center;
+    margin-top: 10px;
+    color: rgb(218, 60, 40);
+}
+
+h3 {
+    margin: 20px 0px 10px 0;
+}
+
+.form-1 {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+}
+
+.form-1 input {
+    text-align: center;
+    padding: 15px;
+    border: none;
+    font-size: 18px;
+}
```

Comment:
`form-1`  is quite a generic name.  If I am reading the CSS and I'm trying to remember "Which one was `form-1` and which one was `form-2`, then a name like `form-quantity` could be helpful?